ryvencore-qt/ryvencore_qt/src/flows/FlowView.py

```diff
@@ -360,38 +408,46 @@ def _redo_activated(self):
         self._undo_stack.redo()
         self.viewport().update()
 
+    def mouseDoubleClickEvent(self, event):
+        # Only allow double clicking over an item
+        if self.itemAt(event.pos()):
+            QGraphicsView.mousePressEvent(self, event)
```

Comment:
why?

Reply:
If I remember correctly, double-clicking with the right button without this would de-select the current selection. I thought that double clicking on the view would be pointless either way, so I added it without further thought.

Reply:
In my mind that's the desired behavior, if they double click on a node while multi selecting, the selection should change, no?

Reply:
That is still the behavior with the left click. If you remove that part, then the right double click also deselects.

Reply:
Oh you meant multi-selecting and double-clicking with the right click. I guess we'd want that. But do we want the selection to be cleared when right double clicking on the view?

Reply:
> if they double click on a node while multi selecting 

Multi-selection only ends when a right or middle click is released.